src/defaultTheme.js

```diff
@@ -1,7 +1,15 @@
 // @flow
 import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
+const mediaQueries = {
+  widthMediaQueryMediumMobile: 414,
+  widthMediaQueryLargeMobile: 576,
+  widthMediaQueryTablet: 768,
+  widthMediaQueryDesktop: 992,
+  widthMediaQueryLargeDesktop: 1200,
+};
+
 export default {
-  orbit: defaultTokens,
+  orbit: { ...defaultTokens, ...mediaQueries },
```

Comment:
Why this isn't  part of `orbit-design-tokens` ? also rather this  way 🙏 

```
Object.assign({}, defaultTokens, mediaQueries}
```

Reply:
Just for test purposes, we need to add it to tokens first 🙂